geode-wan/src/test/java/org/apache/geode/internal/cache/wan/wancommand/WanCommandListDUnitTest.java

```diff
@@ -230,41 +251,39 @@ public void testListGatewaySenderGatewayReceiver() throws GfJsonException {
-    props.setProperty(MCAST_PORT, "0");
-    props.setProperty(DISTRIBUTED_SYSTEM_ID, "1");
-    props.setProperty(LOCATORS, "localhost[" + lnPort + "]");
-    setUpJmxManagerOnVm0ThenConnect(props);
+    Integer lnPort = locatorSite1.getPort();
+    Integer nyPort = locatorSite2.getPort();
 
-    Integer nyPort = vm2.invoke(() -> createFirstRemoteLocator(2, lnPort));
+    // setup servers in Site #1 (London)
+    server1 = startServerWithGroups(3, "Serial_Sender, Parallel_Sender", lnPort);
+    server2 = startServerWithGroups(4, "Serial_Sender, Parallel_Sender", lnPort);
+    server3 = startServerWithGroups(5, "Parallel_Sender, Receiver_Group", lnPort);
 
-    vm6.invoke(() -> createAndStartReceiver(nyPort));
+    // server in Site 2 (New York)
+    server4 = locatorServerStartupRule.startServerVM(6, nyPort);
+    server5 = locatorServerStartupRule.startServerVM(7, nyPort);
 
-    vm3.invoke(() -> createCacheWithGroups(lnPort, "Serial_Sender, Parallel_Sender"));
-    vm3.invoke(() -> createSender("ln_Serial", 2, false, 100, 400, false, false, null, false));
-    vm3.invoke(() -> createSender("ln_Parallel", 2, true, 100, 400, false, false, null, false));
+    server4.invoke(() -> createAndStartReceiver(nyPort));
 
-    vm4.invoke(() -> createCacheWithGroups(lnPort, "Serial_Sender, Parallel_Sender"));
-    vm4.invoke(() -> createSender("ln_Parallel", 2, true, 100, 400, false, false, null, false));
-    vm4.invoke(() -> createSender("ln_Serial", 2, false, 100, 400, false, false, null, false));
+    server1.invoke(() -> createSender("ln_Serial", 2, false, 100, 400, false, false, null, false));
+    server1.invoke(() -> createSender("ln_Parallel", 2, true, 100, 400, false, false, null, false));
 
-    vm5.invoke(() -> createAndStartReceiverWithGroup(lnPort, "Parallel_Sender,Receiver_Group"));
-    vm5.invoke(() -> createSender("ln_Parallel", 2, true, 100, 400, false, false, null, false));
+    server2.invoke(() -> createSender("ln_Parallel", 2, true, 100, 400, false, false, null, false));
+    server2.invoke(() -> createSender("ln_Serial", 2, false, 100, 400, false, false, null, false));
 
+    server3.invoke(() -> createAndStartReceiver(lnPort));
+    server3.invoke(() -> createSender("ln_Parallel", 2, true, 100, 400, false, false, null, false));
 
-    vm7.invoke(() -> createCache(nyPort));
-    vm7.invoke(() -> createSender("ln_Serial", 1, false, 100, 400, false, false, null, false));
-    vm7.invoke(() -> createSender("ln_Parallel", 1, true, 100, 400, false, false, null, false));
+    server5.invoke(() -> createSender("ln_Serial", 1, false, 100, 400, false, false, null, false));
+    server5.invoke(() -> createSender("ln_Parallel", 1, true, 100, 400, false, false, null, false));
 
     pause(10000);
     String command = CliStrings.LIST_GATEWAY + " --" + CliStrings.GROUP + "=Serial_Sender";
-    CommandResult cmdResult = executeCommand(command);
+    CommandResult cmdResult = gfsh.executeCommand(command);
```

Comment:
I think this pattern could be replaced with gfsh.executeAndVerifyCommand.

As it's written here, if the result is not OK the test will just fail with an assertion saying "OK" does not equal "FAILED" but it won't have the command output. If you use gfsh.executeAndVerifyCommand I think it will include the output in the assertion failure.

Reply:
Jared just added a new way of executing gfsh commands and asserts status and output:
gfsh.executeAndAssertThat(command)
   .statusIsSuccess()
   .containsOutput(output...)
   .doesNotContainOutput(...)

Reply:
I think the premise of these JIRAs is to move to the new test rules.  I can look into modifying the command execution and status checks, but it seems outside the bounds of the original JIRA.  Will discuss with Dan.